internal/controller/clusterinstance_validator_test.go

```diff
@@ -126,13 +126,13 @@ var _ = Describe("validateSiteConfig", func() {
 		clusterTemplate, nodeTemplate, extraManifest *corev1.ConfigMap
 		extraManifestName                            = "extra-manifest"
 		nodeTemplateRef                              = "node-template-ref"
-		siteConfig                                   *v1alpha1.SiteConfig
+		clusterInstance                              *v1alpha1.ClusterInstance
```

Comment:
Test name block is still `Describe("validateSiteConfig"`

Reply:
This is addressed in the 2nd commit (8cfb719e1bb732170c85151d270c2d00217c1ecf).